reference/configuration/framework.rst

```diff
@@ -809,6 +811,17 @@ This determines the number of seconds after which data will be seen as "garbage"
 and potentially cleaned up. Garbage collection may occur during session
 start and depends on `gc_divisor`_ and `gc_probability`_.
 
+use_strict_mode
+...............
+
+**type**: ``boolean`` **default**: ``0``
```

Comment:
since this is a boolean node having `0` will cause more troubles so I would indeed change it :)